pkg/helm/index.go

```diff
@@ -71,6 +71,12 @@ func ChartsFromIndex(contents []byte, r *models.Repo, shallow bool) ([]models.Ch
 		// almost 200 chart versions going all the way back many years to version "2.1.4".
 		// So for now, let's just keep track of the latest, not to overwhelm the caller with
 		// all these outdated versions
+
+		// skip if the entry is empty
+		if len(entry) < 1 {
+			continue
+		}
```

Comment:
Perhaps we can add here a log trace like:
```go
		if len(entry) < 1 {
			log.Infof("skipping empty chart: [%s]", key)
			continue
		}
```

The `for` should include the `key` in this case

Reply:
Yep, I'll add the unit test.
I'll also add the log message but we have not any info about the entry, if it is ok I could log only the skipping string :)

Reply:
Great, thanks! I just tried replacing the `for` statement by `for key, entry := range index.Entries {`. In your example (below), I'm getting the key name (in this case, `selenium`). Anyway, no strong opinion here, it's just an info log message.

```yaml
entries:
  selenium: []
  selenium3:
```

Reply:
Thanks for the hint.
It should be done with the last commit :)